tasks/c/medium/read_csv.c

```diff
@@ -6,5 +6,24 @@
 
 int main() {
     // TODO: Implement the program that reads and displays the contents of data.csv file
+    char filename="data.csv";
```

Comment:
The filename is going to be array of characters so this should be either `char filename[]` or a pointer to first character on the array like this `char *filename`.